lib/main.dart

```diff
@@ -13,12 +14,14 @@ late String signingKeyFound = "";
 
 Future<void> appInit() async {
   WidgetsFlutterBinding.ensureInitialized();
-  // try {
-  //   signingKeyFound = await getSigningKey() ?? "unknown";
-  // } catch (e) {
-  //   signingKeyFound = e.toString();
-  // }
   await initializeBaseStoragePath();
+  if (config.initialSetupComplete == false) {
+    final oldSecureStorage = await secureStorage.readAll();
+    final date = DateTime.now().toIso8601String();
+    config.oldSecureStorage[date] = oldSecureStorage;
+    config.save();
+    await secureStorage.deleteAll();
+  }
```

Comment:
this is for debugging only right?

Reply:
No this is for iOS - as it doesn't properly cleanup secure storage.

Reply:
but I don't see any place where you set it to back false
or am I missing the purpose of this, can you explain it

Reply:
It is false by default, after user has completed initial setup it is set to true. Config file is located in normal app directory, that gets removed when user resets app data / reinstalls the app.